Running npm run setup

Just so you know i got this error when i ran npm run setup, but was able to get the database going by creating the formula_1 database and running knex afterwards.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! server-node@1.0.0 setup: `sudo -u postgres psql -c 'CREATE USER f1 WITH SUPERUSER CREATEDB --no-password'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the server-node@1.0.0 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

--- a/package.json
+++ b/package.json
@@ -5,7 +5,8 @@
   "main": "server.js",
   "scripts": {
     "test": "NODE_ENV=test mocha --exit",
-    "database": "psql -c 'CREATE DATABASE formula_1' && psql -c 'CREATE DATABASE formula_1_test'"
+    "database": "psql -c 'CREATE DATABASE formula_1' && psql -c 'CREATE DATABASE formula_1_test'",
+    "seed": "NODE_ENV=test knex migrate:latest && NODE_ENV=test knex seed:run"
   },
   "repository": {
     "type": "git",

--- a/package.json
+++ b/package.json
@@ -5,7 +5,6 @@
   "main": "server.js",
   "scripts": {
     "test": "NODE_ENV=test mocha --exit",
-    "setup": "sudo potato psql -c 'CREATE USER f1 WITH SUPERUSER CREATEDB --no-password'",
     "database": "psql -c 'CREATE DATABASE formula_1' && psql -c 'CREATE DATABASE formula_1_test'"
   },
   "repository": {

--- a/server.js
+++ b/server.js
@@ -10,7 +10,7 @@ app.locals.title = "BYOBE Database";
 app.locals.drivers = [];
 app.locals.teams = [];
 
-app.use(bodyParser.json(), cors);
+app.use(bodyParser.json(), cors());
 app.use(express.static("public"));
 
 app.set("port", process.env.PORT || 3000);

